frontend/app/views/spree/checkout/_payment.html.erb

```diff
@@ -59,7 +59,15 @@
     <p class='field' data-hook='coupon_code'>
       <%= form.label :coupon_code %><br />
       <%= form.text_field :coupon_code %>
+      <button type="button" class="button" id="coupon-code-apply-button">
+        <%= Spree.t(:apply_code) %>
+      </button>
     </p>
+    <div id='coupon_status'
+      data-success-class="success"
```

Comment:
Since these values match the defaults here https://github.com/solidusio/solidus/pull/1090/files#diff-a065997c80be8446e236252c98a12800R7 we don't need to duplicate them here.